eidas_node/datamodels.py

```diff
@@ -128,7 +131,8 @@ def export_xml(self) -> Element:
         self.validate()
         if not self.ROOT_ELEMENT:
             raise TypeError('XMLDataModel subclasses must define ROOT_ELEMENT class attribute.')
-        root = Element(self.ROOT_ELEMENT)
+        root_nsmap = {} if not self.ROOT_NS else {None: self.ROOT_NS}
+        root = Element(self.ROOT_ELEMENT, nsmap=root_nsmap)
```

Comment:
This combined with `ROOT_NS = 'http://cef.eidas.eu/LightResponse'` and `ROOT_NS = 'http://cef.eidas.eu/LightRequest'` is a breaking change, so we must specify that only eIDAS Node 2.5+ is supported in README.md.

Reply:
eIDAS Node 2.4 apparently ignores namespace in LightRequest/LightResponse so this is not breaking change.

Reply:
Glad to hear that :-)